components/dialog/overview.md

```diff
@@ -53,6 +53,12 @@ The Dialog component and its [predefined options]({%slug dialog-predefined%}) ai
 
 Predefined Dialogs are styled substitutes to the standard browser dialogs - confirm, alert and prompt. [Read more about the Blazor Predefined Dialogs]({%slug dialog-predefined%}).
 
+## Integration with other components
+
+You can implement other components (such as [`Filter`]({%slug filter-overview%}), [`CheckBox`]({%slug checkbox-overview%})) as `DialogContent`. In this case, the two-way binding for the `Value` of these components, will not work as expected. The Dialog needs to [`Refresh`](#dialog-reference-and-methods) to reflect UI changes. You can invoke the Dialog's `Refresh` method:
```

Comment:
The information applies to any component and any parameter, is the description should sound more general.

```suggestion
You can use other components inside the `DialogContent`. In this case, the two-way binding for the parameters of these components will not work as expected. The Dialog needs to [`Refresh`](#dialog-reference-and-methods) to reflect UI changes. You can invoke the Dialog's `Refresh` method:
```